internal/pkg/term/prompt/select.go

```diff
@@ -36,7 +38,7 @@ func (o Option) String() string {
 }
 
 // SelectOption prompts the user to select one option from options and returns the Value of the option.
-func (p Prompt) SelectOption(message string, opts []Option, promptCfgs ...PromptConfig) (value string, err error) {
+func (p Prompt) SelectOption(message, help string, opts []Option, promptCfgs ...PromptConfig) (value string, err error) {
```

Comment:
👍 Let's keep the help, but my thought was that most of the time we will leave it "" and the `Hint` field is enough part of `Option`

Reply:
I added `help` because it was needed in `storage init` where we ask for storage type! The help string was useful so I wanted to include it there.